.changesets/feat_geal_implement_redis_connection_pooling.md

```diff
@@ -0,0 +1,5 @@
+### Support Redis connection pooling ([PR #5942](https://github.com/apollographql/router/pull/5942))
+
+This implements Redis connection pooling, for APQ, query planner and entity cache Redis usage. This can improve performance when there is some contention on the Redis connection, or some latency in Redis calls.
```

Comment:
```suggestion
The router now supports Redis connection pooling for APQs, query planners and entity caches. This can improve performance when there is contention on Redis connections or latency in Redis calls.
```